### PR TITLE
Pre-pull images in Minikube

### PR DESCRIPTION
**Pull request by kashifest**
This PR pre-pulls images in minikube to reduce the baremetal-operator pod-initialization time.

### CODE REVIEW

File: lib/common.sh [hunk 1]
```diff
@@ -62,10 +62,14 @@ export SUSHY_TOOLS_IMAGE=${SUSHY_TOOLS_IMAGE:-"quay.io/metal3-io/sushy-tools"}
 
 # Ironic vars
 export IPA_DOWNLOADER_IMAGE=${IPA_DOWNLOADER_IMAGE:-"quay.io/metal3-io/ironic-ipa-downloader:master"}
-export IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metal3-io/ironic:master"}
+export IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metal3-io/ironic"}
+export IRONIC_INSPECTOR_IMAGE=${IRONIC_INSPECTOR_IMAGE:-"quay.io/metal3-io/ironic-inspector"}
```
Comment (russellb) [inline lib/common.sh]: Can you retain the `master` tag, just so we're explicit about which git branch we're following?

Comment (russellb) [reply lib/common.sh]: OK, good point.  If that's what the operator yaml is using then let's keep them consistent.  You can leave it as is.

**Comment by russellb** [comment]
lgtm, but another set of eyes from @hardys would be good

**Comment by maelk** [comment]
lgtm to me too

**Comment by kashifest** [comment]
> lgtm but now needs a rebase
> 
> Another nice enhancement would be the ability to cache images in a local registry and/or build them from source automatically - @derekhiggins added something to do that in https://github.com/openshift-metal3/dev-scripts/blob/master/04_setup_ironic.sh#L19 which can perhaps be refactored to work with both docker and podman then moved to metal3-dev-env.

Thanks @hardys . The rebase is done. I will check the link you provided

**Comment by hardys** [comment]
> Thanks @hardys . The rebase is done. I will check the link you provided

Thanks!  The link was just FYI - there are two parts we might consider, first is to configure docker-distribution on the test host (which could be configured as a pull-through cache to avoid repeatedly downloading the container images on the ephemeral minikube vm)

The second is to automatically build images from git and push to the local registry, which would be useful for testing the ironic images etc.  I'll try to spend some time looking at if we can easily reuse some of what has already been done in the dev-scripts repo.

**Pull request merged**